README.md

```diff
@@ -1,35 +1,33 @@
 # Federation Dev
-This repository is a place holder for various demonstrations, labs, and examples
-of the use of KubeFed.
+This repository contains demonstrations, labs, and examples
+of the use of Kubefed and/or the management of multiple OpenShift clusters.
 
 ## What is Federation
-Put quite simply, Federation is the process of connecting one or more
-Kubernetes clusters. KubeFed is the Kubernetes project supporting the concept
-of Federation on Kubernetes clusters. The explanation from OperatorHub states
-
+Put quite simply, Federation is the process of connecting one or more Kubernetes clusters. The explanation from OperatorHub states
 ``
 Kubernetes Federation is a tool to sync (aka "federate") a set of Kubernetes objects from a "source" into a set of other clusters. Common use-cases include federating Namespaces across all of your clusters or rolling out an application across several geographically distributed clusters. The Kubernetes Federation Operator runs all of the components under the hood to quickly get up and running with this powerful concept. Federation is a key part of any Hybrid Cloud capability.
 ``
-
 There are two ways to use Kubefed currently: Namespace and Cluster scoped.
 
+### RHTE lab
+If you are attend Red Hat tech exchange follow the link below will bring you to the first lab.<br>
+[Red Hat Tech Exchange Lab - Hands on with Red Hat Multi-Cluster Federation: Application Portability](./labs/1.md)<br>
```

Comment:
I think this needs to point a clean TOC page that lists all the labs.  then, lgtm.

Reply:
@scollier give it a look now